### PR TITLE
Random pictures

### PR DESCRIPTION
**Pull request by wildcoder51**
API's call is functionnal. We can display the picture of Hubble by the API.
CSS ok. PR before add selection system and display full screen.

### CODE REVIEW

File: picspace/src/Components/RandomPictures.js [hunk 1]
```diff
@@ -1,21 +1,63 @@
...
+      name : "",
+      file_url: "",
+      arrayId: [],
+      arrayPictures:[],
+    }
+  this.getIdPicture = this.getIdPicture.bind(this);
+  }
+ 
+  getIdPicture() {
+    var tempArray = [];
```
Comment (ghost) [inline picspace/src/Components/RandomPictures.js]: No more var

File: picspace/src/Components/RandomPictures.js [hunk 2]
```diff
@@ -1,21 +1,63 @@
...
+      // Extract the DATA from the received response
+      .then(response => {
+        for(var i = 0; i < response.data.length; i++) {
+          this.state.arrayId.push(response.data[i].id);
+        };
+       
+        for(var y = 0; y < this.state.arrayId.length; y++) {
+          axios.get('http://hubblesite.org/api/v3/image/'+this.state.arrayId[y])
+          .then(response => {
+            if (response.data.image_files !== undefined) {
```
Comment (ghost) [inline picspace/src/Components/RandomPictures.js]: You do not need to compare to undefined

File: picspace/src/Components/RandomPictures.js [hunk 3]
```diff
@@ -1,21 +1,63 @@
...
+  }
+ 
+  getIdPicture() {
+    var tempArray = [];
+    // Send the request
+    axios.get('http://hubblesite.org/api/v3/images/all')
+      // Extract the DATA from the received response
+      .then(response => {
+        for(var i = 0; i < response.data.length; i++) {
+          this.state.arrayId.push(response.data[i].id);
```
Comment (ghost) [inline picspace/src/Components/RandomPictures.js:26]: Use es6 template string

File: picspace/src/Components/RandomPictures.js (hunk 1 above)
Comment (wildcoder51) [reply picspace/src/Components/RandomPictures.js]: i changed the type of variable (var > let).
i used ES6 synthax for url.

File: picspace/src/Components/RandomPictures.js [hunk 4]
```diff
@@ -1,21 +1,79 @@
...
+      name : "",
+      file_url: "",
+      arrayId: [],
+      arrayPictures:[],
+      //onSelect : false,
+      isSelected : false,
+      arrayBasket : [],
+    }
+    this.getIdPicture = this.getIdPicture.bind(this);
+    //this.handleClick = this.handleClick.bind(this);
```
Comment (ghost) [inline picspace/src/Components/RandomPictures.js]: Dead code

File: picspace/src/Components/RandomPictures.js [hunk 5]
```diff
@@ -1,21 +1,79 @@
...
+
+  componentDidMount() {
+    this.getIdPicture()
+  }
+
+  // handleClick = (onSelect) =>{
+  //   this.setState({
+  //     onSelect : !this.state.onSelect,
+  //   })
+  // }
```
Comment (ghost) [inline picspace/src/Components/RandomPictures.js]: dead code

**Review by ghost — APPROVED** [review]

**Pull request merged**